cmds/exp/nvme_unlock/nvme_unlock.go

```diff
@@ -69,6 +70,20 @@ func getSysfsInfo(index string, field string) (string, error) {
 }
 
 func run(disk string, verbose bool, verboseNoSanitize bool, noRereadPartitions bool, lock bool) error {
+	if disk == "" {
+		disks, err := finddrive.FindSlotType(finddrive.M2MKeySlotType)
+		if err != nil {
+			return fmt.Errorf("error finding boot disk: %v", err)
+		}
+		if len(disks) == 0 {
+			return fmt.Errorf("no boot disk found")
+		}
+		disk = disks[0]
+		if len(disks) > 1 {
+			log.Printf("Multiple boot disk candidates found, using the first from the list: %v", disks)
+		}
```

Comment:
Do you think it'd be helpful to print sth like: 

boot disk is discovered, and it is: %v  (you will likely rephrase it better than mine

Reply:
The disk name is printed at the end of run(), but I added another log for it here gated under the verbose flag.

Reply:
It's already printed at the end of the command, but I added a verbose-gated log to print it here (in case the command is failing before that point).